src/som/interpreter/actors/EventualMessage.java

```diff
@@ -20,6 +21,7 @@
   protected final Object[]  args;
   protected final SResolver resolver;
   protected final RootCallTarget onReceive;
+  protected final EventualMessage causalMessage;
   protected final long causalMessageId;
```

Comment:
ok. @daumayr can we move the `causalMessageId` into the `causalMessage`? That could also avoid that it is looked up multiple times via `EventualMessage.getCurrentExecutingMessageId()`? Might be a small performance win even?